src/main/java/org/folio/notes/domain/repository/NoteRepository.java

```diff
@@ -39,11 +37,7 @@ static Specification<NoteEntity> domainEq(String domain) {
   }
 
   static Specification<NoteEntity> contentLike(String text) {
-    return (root, query, cb) -> new IlikePredicate(
-      (CriteriaBuilderImpl) cb,
-      root.get(NoteEntity_.indexedContent),
-      "%" + text + "%"
-    );
+    return (root, query, cb) -> cb.like(root.get(NoteEntity_.indexedContent), "%" + text + "%");
```

Comment:
Please return the previous logic. It should be case-insensitive

Reply:
Done